reference/forms/types/options/choice_value.rst.inc

```diff
@@ -1,28 +1,20 @@
 ``choice_value``
 ~~~~~~~~~~~~~~~~
 
-**type**: ``callable`` or ``string`` **default**: ``null``
+**type**: ``callable``, ``string`` or :class:`Symfony\\Component\\PropertyAccess\\PropertyPath` **default**: ``null``
 
 Returns the string "value" for each choice, which must be unique across all choices.
 This is used in the ``value`` attribute in HTML and submitted in the POST/PUT requests.
 You don't normally need to worry about this, but it might be handy when processing
 an API request (since you can configure the value that will be sent in the API request).
 
-This can be a callable or a property path. If ``null`` is given, an incrementing
-integer is used as the value.
+This can be a callable or a property path. By default, the choices are used if they
+can be casted to strings. Otherwise an incrementing integer is used.
```

Comment:
Should we mention if the autoincrementing integer starts in `0` or `1`?

Reply:
`0`